iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -667,6 +667,21 @@ - (void)updateBookmark:(MWMMarkID)bookmarkId
     bookmark->SetCustomName(title.UTF8String);
 }
 
+- (void)updateBookmark:(MWMMarkID)bookmarkId
+              setColor:(MWMBookmarkColor)color {
+  auto editSession = self.bm.GetEditSession();
+
+  auto bookmark = editSession.GetBookmarkForEdit(bookmarkId);
+  if (!bookmark)
+    return;
+
+  auto kmlColor = convertBookmarkColor(color);
```

Comment:
convertBookmarkColor is not clear. A better name hints what is converted to what.